### PR TITLE
Addes the option of banaction

### PR DESCRIPTION
**Pull request by stonewalljones**
This change allow the ability to set the ban action for RHEL/CentOS 7 (Manipo) systems

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -30,6 +30,7 @@
   $service_enable           = true,
 
   $action                   = 'action_mb',
+  $banaction                = 'iptables-multiport',
```
Comment (bastelfreak) [inline manifests/init.pp]: please add a datatype for the new parameter

**Comment by bastelfreak** [comment]
Hi @stonewalljones, thanks for the PR. This module got recently migrated to Vox Pupuli. We require a datatype for new parameters. Can you please add it and rebase?

**Comment by danifr** [comment]
I'm interested on this PR. I can pick it up from here if the OP is does not have time to work on it anymore.

**Comment by bastelfreak** [comment]
thanks for your interest @danifr ! You can checkout the branch, push it to your fork and resubmit it as a new pull request.

**Comment by danifr** [comment]
Hi Tim, I'm afraid this feature was added already here https://github.com/voxpupuli/puppet-fail2ban/commit/e14b2933511417262195087f7c774a872849665c.
I'll try it out and open a new MR in case there are some missing bits.

I think we can close this one :)

**Comment by bastelfreak** [comment]
ah alright, thanks for the info.

**Pull request closed without merging**